# in cm
point_1:
  x: 200
  y: 100
point_2:
  x: 200.0
  y: -250.0
point_3:
  x: -100.0
  y: 50.0
point_4:
  x: -200.0
  y: 250.0
point_5:
  x: 100
  y: 0
